src/app/components/site/Homepage.tsx

```diff
@@ -71,6 +71,70 @@ export const Homepage = () => {
           </section>
         )}
 
+        {!user?.loggedIn && (
+          <section id="resources" className="resources-section">
+            <Container className="pt-4 pb-5">
+              <div className="pt-5">
+                <h2 className="homepage-sub-title text-left mb-4">Resources</h2>
+              </div>
+              <Row>
+                <div className="columns-container">
+                  <Col className="custom-col-60">
+                    <div className="resources-background">
+                      <p className="resources-sub-heading">Computer science topics</p>
+                      <p className="resources-text-description">
+                        {
+                          "Discover our free computer science topics and questions, mapped to the leading exam specifications in England. Review key curriculum areas and get specific feedback to address common misconceptions."
+                        }
+                      </p>
+                      <div className="center-container">
+                        <Link className="resources-links" to="/topics/gcse">
+                          GCSE topics
+                        </Link>
+                        <Link className="resources-links" to="/topics/a_level">
+                          A level topics
+                        </Link>
+                      </div>
+                    </div>
+                  </Col>
+                  <Col className="custom-col-30">
+                    <div className="resources-background">
+                      <div className="resources-sub-heading-container">
+                        <p className="resources-sub-heading">Question finder</p>
+                        <div className="tooltip-container">
+                          <img src={questionFinderSVG} alt="Question Finder" className="q-finder-image" />
+                          <span className="tooltip-text">
+                            You can build a gameboard by selecting the areas of interest, stage and difficulties.
```

Comment:
trivial: do we have any static content spaces in the codebase where this sort of stuff could live? 

If not, my above suggestion of making a component for this then passing that content through would still work nicely

Reply:
We don't have a place where all static content lives. I have always seen content placed within the files. 
Maybe this is something we can add to the back log?

Reply:
potentially. let me think on it :)